fenix/app/src/androidTest/java/org/mozilla/fenix/onboarding/view/JunoOnboardingMapperTest.kt

```diff
@@ -19,28 +22,35 @@ class JunoOnboardingMapperTest {
     val activityTestRule =
         HomeActivityIntentTestRule.withDefaultSettingsOverrides(skipOnboarding = true)
 
+    private lateinit var context: Context
+
+    @Before
+    fun setup() {
+        context = TestHelper.appContext
+    }
+
     @Test
     fun showNotificationTrue_showAddWidgetFalse_pagesToDisplay_returnsSortedListOfAllConvertedPages_withoutAddWidgetPage() {
         val expected = listOf(defaultBrowserPageUiData, syncPageUiData, notificationPageUiData)
-        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(true, false))
+        assertEquals(expected, unsortedAllKnownCardData.toPageUiData(context, true, false))
```

Comment:
Out of scope, but while we're here, could we add the args for the boolean args to remove the `Boolean literal argument without a parameter name ` warning?